recipes/pyvitae/meta.yaml

```diff
@@ -0,0 +1,122 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
As it says here, please remove comments not specific to the package you are submitting here.